spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonSerializer.java

```diff
@@ -135,4 +155,11 @@ public void close() {
 		// No-op
 	}
 
+	private boolean isTypeMapperExplicitlySet() {
+		return this.typeMapperExplicitlySet;
+	}
+
+	private void setTypeMapperExplicitlySet(boolean typeMapperExplicitlySet) {
```

Comment:
Sorry, not clear why do we need these two method?
You didn't introduce them in the `JsonDeserializer`, why here then?
Thanks

Reply:
Sorry, my mistake - removed